seedcase_sprout/core/sprout_checks/data/get_pandera_checks.py

```diff
@@ -0,0 +1,152 @@
+import pandera.polars as pa
+from xmlschema.names import (
+    XSD_BASE64_BINARY,
+    XSD_DATE,
+    XSD_DATETIME,
+    XSD_DURATION,
+    XSD_GYEAR,
+    XSD_GYEAR_MONTH,
+    XSD_TIME,
+)
+
+from seedcase_sprout.core.properties import FieldProperties
+from seedcase_sprout.core.sprout_checks.data.data_type_checks import (
+    check_email,
+    check_geopoint,
+    check_json,
+    check_uuid,
+    check_xml_type,
+)
+
+# https://datapackage.org/standard/table-schema/#boolean
+BOOLEAN_VALUES = {"false", "False", "FALSE", "0", "true", "True", "TRUE", "1"}
+
+STRING_FORMAT_CHECKS = {
+    "email": pa.Check(
+        check_email,
+        element_wise=True,
+        error="Invalid email address.",
+    ),
+    "binary": pa.Check(
+        lambda value: check_xml_type(value, XSD_BASE64_BINARY),
+        element_wise=True,
+        error="Invalid data format. Expected a Base64-encoded string.",
+    ),
+    "uuid": pa.Check(
+        check_uuid,
+        element_wise=True,
+        error="Invalid UUID format. Expected a valid UUID.",
+    ),
+}
+
+
+def get_pandera_checks(field: FieldProperties) -> list[pa.Check]:
+    """Returns the Pandera checks appropriate for the field's format and data type.
+
+    Args:
+        field: The field to get the checks for.
+
+    Returns:
+        The appropriate Pandera checks.
+    """
+    if field.type == "string" and (
+        format_check := STRING_FORMAT_CHECKS.get(field.format, None)
+    ):
+        return [format_check]
+
+    match field.type:
+        case "boolean":
+            return [
+                pa.Check(
+                    lambda value: value in BOOLEAN_VALUES,
+                    element_wise=True,
+                    error=f"Invalid boolean value. Must be one of {BOOLEAN_VALUES}.",
```

Comment:
```suggestion
                    error=f"The logic/boolean value given is {value}, but needs to be one of {BOOLEAN_VALUES}.",
```

Is it possible to add the value into the error message?

Reply:
Same as above, Pandera will add failure cases